biojava-structure/src/main/java/org/biojava/nbio/structure/basepairs/BasePairParameters.java

```diff
@@ -0,0 +1,718 @@
+        return stepParameters[bp][3];
+    }
+
+    /**
+     * Return the slide for the given base pair, relative to the one before it.
+     * @param bp the number of the base pair (starting with 0)
+     * @return the value as a double (in Å)
+     */
+    public Double getSlide(int bp) {
+        if (bp < 0 || bp >= getStepParameters().length) throw new IllegalArgumentException();
+        return stepParameters[bp][4];
+    }
+
+    /**
+     * Return the rise for the given base pair, relative to the one before it.
+     * @param bp the number of the base pair (starting with 0)
+     * @return the value as a double (in Å)
+     */
+    public Double getRise(int bp) {
+        if (bp < 0 || bp >= getStepParameters().length) throw new IllegalArgumentException();
+        return stepParameters[bp][5];
+    }
+
+
+    /**
+     * This reports all the nucleic acid chains and has an option to remove duplicates if you
+     * are considering an analysis of only unique DNA or RNA helices in the Structure.
+     * @param removeDups If true, it will ignore duplicate chains
+     * @return A list of all the nucleic acid chains in order of the Structure
+     */
+    public List<Chain> getNucleicChains(boolean removeDups) {
+        if (structure == null) return new ArrayList<>();
+        List<Chain> chains = structure.getChains();
+        List<Chain> result = new ArrayList<>();
+        for (Chain c: chains) {
+            if (c.isNucleicAcid()) {
+                result.add(c);
+            }
+        }
+        if (removeDups) for (int i = 0; i < result.size(); i++) {
+            for (int j = i+2; j < result.size(); j++) {
+                // remove duplicate sequences (structures with two or more identical units)
+                if (result.get(i).getAtomSequence().equals(result.get(j).getAtomSequence())) {
+                    result.remove(j);
+                }
+            }
+        }
+        return result;
+    }
+
+    /**
+     * This performs a search for base pairs in the structure.  The criteria is alignment of
+     * sequences and the canonical base pairs of DNA or RNA. Use MismatchedBasePairParameters
+     * or TertiaryBasePairParameters for finding higher-order associations.
+     * @param chains The list of chains already found to be nucleic acids
+     * @return The list of corresponding Watson-Crick groups as pairs, as a Pair of nucleic acid Groups
+     */
+    public List<Pair<Group>> findPairs(List<Chain> chains) {
+        List<Pair<Group>> result = new ArrayList<>();
+        for (int i = 0; i < chains.size(); i++) {
+            Chain c = chains.get(i);
+            for (int j = i+1; j < chains.size(); j++) {
+                String complement = complement(chains.get(j).getAtomSequence(), useRNA);
+                String match = longestCommonSubstring(c.getAtomSequence(), complement);
+                //log.info(c.getAtomSequence() + " " + chains.get(j).getAtomSequence() + " " + match);
```

Comment:
Rather than comment out this log statement, you could leave it in and use a lower log level, e.g.
```
if (log.isDebugEnabled()) {
  log.debug(...)
}
```